packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -166,6 +166,13 @@ export enum EnqueueStrategy {
  * @returns Promise that resolves to {@apilink BatchAddRequestsResult} object.
  */
 export async function enqueueLinks(options: SetRequired<EnqueueLinksOptions, 'requestQueue' | 'urls'>): Promise<BatchAddRequestsResult> {
+    if (!options || Object.keys(options).length === 0) {
+        throw new RangeError([
+            `enqueueLinks() was called without the required options. You might have imported it from "crawlee" instead of using the context-aware method.`,
+            'Check out our guide on how to use enqueueLinks() here: https://crawlee.dev/docs/examples/crawl-relative-links',
```

Comment:
This is a tiny detail, but it seems a bit weird to say "you might have imported it". Shouldn't we know that and only throw this error when this is the case?

Reply:
Hmm, I guess for most cases yeah, this error would be thrown only if the user imports it. I can remove that if you think it's redundant

Reply:
How about this? Sorry for delay. I keep forgetting to check GitHub now that I don't code a lot.

```suggestion
            'enqueueLinks() was called without the required options. You can only do that when you use the `crawlingContext.enqueueLinks()` method in request handlers.',
            'Check out our guide on how to use enqueueLinks() here: https://crawlee.dev/docs/examples/crawl-relative-links',
```